database/database.go

```diff
@@ -3,10 +3,25 @@ package database
 import (
 	"database/sql"
 	"fmt"
+
+	_ "github.com/go-sql-driver/mysql"
```

Comment:
You need to update go.mod for this:
```
database/database.go:8:2: no required module provides package github.com/mattn/go-sqlite3; to add it:
	go get github.com/mattn/go-sqlite3
```

Reply:
This was accidentally merged in #21, I just rebased the branch so it includes the changes.